src/components/Book.module.css

```diff
@@ -1,10 +1,22 @@
 @import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
 
+.percentageContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+}
+
 .circle {
   width: 80px;
   height: 80px;
 }
 
+.border {
+  border-right: 2px solid rgba(0, 0, 0, 0.2);
+  padding: 30px 30px;
+}
+
 .bookCard {
```

Comment:
- [ ] Please update the color of your entire to : `#3e98c7` for your comment,edit,remove and the nav user icon and for the app logo and the book and categories links, also the update progress button should have a `#3e98c7`  back-ground-color and set the border to none and your result should be like this :point_down: 

![image](https://user-images.githubusercontent.com/75126481/161113308-f408ab4f-8dc8-4c3b-a130-dc27dfbde364.png)


For now it is like this :point_down: 

![image](https://user-images.githubusercontent.com/75126481/161114762-f3108fab-b71a-496a-8ae3-ff7bf53ab168.png)